components/TopHeader.tsx

```diff
@@ -1,26 +1,103 @@
 import React from "react";
 import styles from "../styles/TopHeader.module.css";
+import { Link as ReactScrollLink } from "react-scroll";
+import Image from "next/image";
 
 export default function TopHeader() {
 	return (
 		<div className={styles.globalTopHeader}>
```

Comment:
jeśli ten styl jest już w komponencie pod nazwą TopHeader, to nie powinien mieć nic z global w nazwie